src/core/services/theming/theming.spec.js

```diff
@@ -435,6 +436,7 @@ describe('$mdThemingProvider', function() {
       expect(config2.registeredStyles.length).toBe(0);
       expect(config2.nonce).toBe(null);
       expect(config2.alwaysWatchTheme).toBe(false);
+      expect(config2.appendTo.tagName).toBe(document.head.tagName);
```

Comment:
Can you please add a test that verifies setting `appendTo` to an element works and that the theme's styles are still applied as expected?